### PR TITLE
Resubmit #68682

### LINKED ISSUE #71454: `Logical error: 'Unexpected number of columns in result sample block: 4 instead of 1 (1 + 0 + 0)'.`
https://s3.amazonaws.com/clickhouse-test-reports/71308/c1ce74f52f9b5b53db7bcf43aa0a1a47c9dd9859/ast_fuzzer__ubsan_.html
```
2024.11.04 22:28:17.285792 [ 886 ] {ab3d816f-6cd9-4081-a720-f3d290a55ef8} <Fatal> : Logical error: 'Unexpected number of columns in result sample block: 4 instead of 1 (1 + 0 + 0)'.
2024.11.04 22:28:17.299155 [ 886 ] {ab3d816f-6cd9-4081-a720-f3d290a55ef8} <Fatal> : Stack trace (when copying this message, always include the lines below):

2024.11.04 22:31:41.733487 [ 1407 ] {} <Fatal> BaseDaemon: ########## Short fault info ############
2024.11.04 22:31:41.733517 [ 1407 ] {} <Fatal> BaseDaemon: (version 24.11.1.771, build id: F50B1C74F5571E881A970978ABCE0EFC4B8F2122, git hash: 1ffbf44f318407909ebe25afa572c7094e808654, architecture: x86_64) (from thread 886) Received signal 6
2024.11.04 22:31:41.733527 [ 1407 ] {} <Fatal> BaseDaemon: Signal description: Aborted
2024.11.04 22:31:41.733536 [ 1407 ] {} <Fatal> BaseDaemon: 
2024.11.04 22:31:41.733550 [ 1407 ] {} <Fatal> BaseDaemon: Stack trace: 0x0000557ecd94858d 0x0000557ecddb92f8 0x00007fa88efa2520 0x00007fa88eff69fd 0x00007fa88efa2476 0x00007fa88ef887f3 0x0000557ecd8fd7f1 0x0000557ecd8fdd93 0x0000557ecd8feab7 0x0000557ec1622296 0x0000557edb005937 0x0000557edafdf0c7 0x0000557eddd582db 0x0000557edd714040 0x0000557edd6fed03 0x0000557edd6fdd76 0x0000557edd6fd98a 0x0000557edd71946c 0x0000557ecda32c2f 0x0000557ecda39e2a 0x00007fa88eff4ac3 0x00007fa88f086850
2024.11.04 22:31:41.733580 [ 1407 ] {} <Fatal> BaseDaemon: ########################################
2024.11.04 22:31:41.733636 [ 1407 ] {} <Fatal> BaseDaemon: (version 24.11.1.771, build id: F50B1C74F5571E881A970978ABCE0EFC4B8F2122, git hash: 1ffbf44f318407909ebe25afa572c7094e808654) (from thread 886) (query_id: ab3d816f-6cd9-4081-a720-f3d290a55ef8) (query: SELECT  materialize('key1'),    toNullable(1 AS k) FROM (SELECT number, number, number, number, k, k FROM numbers(5) WHERE toNullable(255)) AS t1 FULL OUTER JOIN (SELECT number, number + 1 AS k, k,        '1', toLowCardinality(1),   1,        toUInt256(1), k, k FROM numbers(3)) AS t2 ON t1.number = t2.number ORDER BY toInt256(255) DESC, 1 DESC NULLS LAST) Received signal Aborted (6)
2024.11.04 22:31:41.733696 [ 1407 ] {} <Fatal> BaseDaemon: 
2024.11.04 22:31:41.733733 [ 1407 ] {} <Fatal> BaseDaemon: Stack trace: 0x0000557ecd94858d 0x0000557ecddb92f8 0x00007fa88efa2520 0x00007fa88eff69fd 0x00007fa88efa2476 0x00007fa88ef887f3 0x0000557ecd8fd7f1 0x0000557ecd8fdd93 0x0000557ecd8feab7 0x0000557ec1622296 0x0000557edb005937 0x0000557edafdf0c7 0x0000557eddd582db 0x0000557edd714040 0x0000557edd6fed03 0x0000557edd6fdd76 0x0000557edd6fd98a 0x0000557edd71946c 0x0000557ecda32c2f 0x0000557ecda39e2a 0x00007fa88eff4ac3 0x00007fa88f086850
2024.11.04 22:31:41.750510 [ 1407 ] {} <Fatal> BaseDaemon: 0.0. inlined from ./build_docker/./src/Common/StackTrace.cpp:380: StackTrace::tryCapture()
2024.11.04 22:31:41.750556 [ 1407 ] {} <Fatal> BaseDaemon: 0. ./build_docker/./src/Common/StackTrace.cpp:349: StackTrace::StackTrace(ucontext_t const&) @ 0x00000000255e458d

2024.11.04 22:31:41.768504 [ 1407 ] {} <Fatal> BaseDaemon: 1. ./build_docker/./src/Common/SignalHandlers.cpp:85: signalHandler(int, siginfo_t*, void*) @ 0x0000000025a552f8
2024.11.04 22:31:41.768539 [ 1407 ] {} <Fatal> BaseDaemon: 2. ? @ 0x00007fa88efa2520
2024.11.04 22:31:41.768581 [ 1407 ] {} <Fatal> BaseDaemon: 3. ? @ 0x00007fa88eff69fd
2024.11.04 22:31:41.768618 [ 1407 ] {} <Fatal> BaseDaemon: 4. ? @ 0x00007fa88efa2476
2024.11.04 22:31:41.768658 [ 1407 ] {} <Fatal> BaseDaemon: 5. ? @ 0x00007fa88ef887f3

2024.11.04 22:31:41.796683 [ 1407 ] {} <Fatal> BaseDaemon: 6. ./build_docker/./src/Common/Exception.cpp:48: DB::abortOnFailedAssertion(String const&, void* const*, unsigned long, unsigned long) @ 0x00000000255997f1
2024.11.04 22:31:41.823891 [ 1407 ] {} <Fatal> BaseDaemon: 7. ./build_docker/./src/Common/Exception.cpp:71: DB::handle_error_code(String const&, int, bool, std::vector<void*, std::allocator<void*>> const&) @ 0x0000000025599d93
2024.11.04 22:31:41.849903 [ 1407 ] {} <Fatal> BaseDaemon: 8. ./build_docker/./src/Common/Exception.cpp:115: DB::Exception::Exception(DB::Exception::MessageMasked&&, int, bool) @ 0x000000002559aab7
2024.11.04 22:31:41.915679 [ 1407 ] {} <Fatal> BaseDaemon: 9. DB::Exception::Exception(PreformattedMessage&&, int) @ 0x00000000192be296
2024.11.04 22:31:41.971866 [ 1407 ] {} <Fatal> BaseDaemon: 10. ./src/Common/Exception.h:128: DB::Exception::Exception<unsigned long, unsigned long&, unsigned long&, unsigned long, unsigned long>(int, FormatStringHelperImpl<std::type_identity<unsigned long>::type, std::type_identity<unsigned long&>::type, std::type_identity<unsigned long&>::type, std::type_identity<unsigned long>::type, std::type_identity<unsigned long>::type>, unsigned long&&, unsigned long&, unsigned long&, unsigned long&&, unsigned long&&) @ 0x0000000032ca1937
2024.11.04 22:31:42.027259 [ 1407 ] {} <Fatal> BaseDaemon: 11. ./build_docker/./src/Interpreters/HashJoin/HashJoin.cpp:1252: DB::HashJoin::getNonJoinedBlocks(DB::Block const&, DB::Block const&, unsigned long) const @ 0x0000000032c7b0c7
2024.11.04 22:31:42.041832 [ 1407 ] {} <Fatal> BaseDaemon: 12. ./build_docker/./src/Processors/Transforms/JoiningTransform.cpp:141: DB::JoiningTransform::work() @ 0x00000000359f42db
2024.11.04 22:31:42.047388 [ 1407 ] {} <Fatal> BaseDaemon: 13.0. inlined from ./build_docker/./src/Processors/Executors/ExecutionThreadContext.cpp:47: DB::executeJob(DB::ExecutingGraph::Node*, DB::ReadProgressCallback*)
2024.11.04 22:31:42.047427 [ 1407 ] {} <Fatal> BaseDaemon: 13. ./build_docker/./src/Processors/Executors/ExecutionThreadContext.cpp:96: DB::ExecutionThreadContext::executeTask() @ 0x00000000353b0040
2024.11.04 22:31:42.061874 [ 1407 ] {} <Fatal> BaseDaemon: 14. ./build_docker/./src/Processors/Executors/PipelineExecutor.cpp:289: DB::PipelineExecutor::executeStepImpl(unsigned long, std::atomic<bool>*) @ 0x000000003539ad03
2024.11.04 22:31:42.075293 [ 1407 ] {} <Fatal> BaseDaemon: 15.0. inlined from ./build_docker/./src/Processors/Executors/PipelineExecutor.cpp:255: DB::PipelineExecutor::executeSingleThread(unsigned long)
2024.11.04 22:31:42.075342 [ 1407 ] {} <Fatal> BaseDaemon: 15. ./build_docker/./src/Processors/Executors/PipelineExecutor.cpp:441: DB::PipelineExecutor::executeImpl(unsigned long, bool) @ 0x0000000035399d76
2024.11.04 22:31:42.090266 [ 1407 ] {} <Fatal> BaseDaemon: 16. ./build_docker/./src/Processors/Executors/PipelineExecutor.cpp:126: DB::PipelineExecutor::execute(unsigned long, bool) @ 0x000000003539998a
2024.11.04 22:31:42.103406 [ 1407 ] {} <Fatal> BaseDaemon: 17.0. inlined from ./build_docker/./src/Processors/Executors/PullingAsyncPipelineExecutor.cpp:83: DB::threadFunction(DB::PullingAsyncPipelineExecutor::Data&, std::shared_ptr<DB::ThreadGroup>, unsigned long, bool)
2024.11.04 22:31:42.103458 [ 1407 ] {} <Fatal> BaseDaemon: 17.1. inlined from ./build_docker/./src/Processors/Executors/PullingAsyncPipelineExecutor.cpp:109: operator()
2024.11.04 22:31:42.103483 [ 1407 ] {} <Fatal> BaseDaemon: 17.2. inlined from ./contrib/llvm-project/libcxx/include/__functional/invoke.h:394: ?
2024.11.04 22:31:42.103512 [ 1407 ] {} <Fatal> BaseDaemon: 17.3. inlined from ./contrib/llvm-project/libcxx/include/tuple:1789: _ZNSt3__118__apply_tuple_implB6v15007IRZN2DB28PullingAsyncPipelineExecutor4pullERNS1_5ChunkEmE3$_0RNS_5tupleIJEEETpTnmJEEEDcOT_OT0_NS_15__tuple_indicesIJXspT1_EEEE
2024.11.04 22:31:42.103543 [ 1407 ] {} <Fatal> BaseDaemon: 17.4. inlined from ./contrib/llvm-project/libcxx/include/tuple:1798: decltype(auto) std::apply[abi:v15007]<DB::PullingAsyncPipelineExecutor::pull(DB::Chunk&, unsigned long)::$_0&, std::tuple<>&>(DB::PullingAsyncPipelineExecutor::pull(DB::Chunk&, unsigned long)::$_0&, std::tuple<>&)
2024.11.04 22:31:42.103569 [ 1407 ] {} <Fatal> BaseDaemon: 17.5. inlined from ./src/Common/ThreadPool.h:311: operator()
2024.11.04 22:31:42.103597 [ 1407 ] {} <Fatal> BaseDaemon: 17.6. inlined from ./contrib/llvm-project/libcxx/include/__functional/invoke.h:394: ?
2024.11.04 22:31:42.103620 [ 1407 ] {} <Fatal> BaseDaemon: 17.7. inlined from ./contrib/llvm-project/libcxx/include/__functional/invoke.h:479: ?
2024.11.04 22:31:42.103646 [ 1407 ] {} <Fatal> BaseDaemon: 17.8. inlined from ./contrib/llvm-project/libcxx/include/__functional/function.h:235: ?
2024.11.04 22:31:42.103670 [ 1407 ] {} <Fatal> BaseDaemon: 17. ./contrib/llvm-project/libcxx/include/__functional/function.h:716: ? @ 0x00000000353b546c
2024.11.04 22:31:42.114097 [ 1407 ] {} <Fatal> BaseDaemon: 18.0. inlined from ./contrib/llvm-project/libcxx/include/__functional/function.h:848: ?
2024.11.04 22:31:42.114125 [ 1407 ] {} <Fatal> BaseDaemon: 18.1. inlined from ./contrib/llvm-project/libcxx/include/__functional/function.h:1197: ?
2024.11.04 22:31:42.114150 [ 1407 ] {} <Fatal> BaseDaemon: 18. ./build_docker/./src/Common/ThreadPool.cpp:785: ThreadPoolImpl<std::thread>::ThreadFromThreadPool::worker() @ 0x00000000256cec2f
2024.11.04 22:31:42.131493 [ 1407 ] {} <Fatal> BaseDaemon: 19.0. inlined from ./contrib/llvm-project/libcxx/include/__functional/invoke.h:359: ?
2024.11.04 22:31:42.131551 [ 1407 ] {} <Fatal> BaseDaemon: 19.1. inlined from ./contrib/llvm-project/libcxx/include/thread:284: void std::__thread_execute[abi:v15007]<std::unique_ptr<std::__thread_struct, std::default_delete<std::__thread_struct>>, void (ThreadPoolImpl<std::thread>::ThreadFromThreadPool::*)(), ThreadPoolImpl<std::thread>::ThreadFromThreadPool*, 2ul>(std::tuple<std::unique_ptr<std::__thread_struct, std::default_delete<std::__thread_struct>>, void (ThreadPoolImpl<std::thread>::ThreadFromThreadPool::*)(), ThreadPoolImpl<std::thread>::ThreadFromThreadPool*>&, std::__tuple_indices<2ul>)
2024.11.04 22:31:42.131591 [ 1407 ] {} <Fatal> BaseDaemon: 19. ./contrib/llvm-project/libcxx/include/thread:295: void* std::__thread_proxy[abi:v15007]<std::tuple<std::unique_ptr<std::__thread_struct, std::default_delete<std::__thread_struct>>, void (ThreadPoolImpl<std::thread>::ThreadFromThreadPool::*)(), ThreadPoolImpl<std::thread>::ThreadFromThreadPool*>>(void*) @ 0x00000000256d5e2a
2024.11.04 22:31:42.131621 [ 1407 ] {} <Fatal> BaseDaemon: 20. ? @ 0x00007fa88eff4ac3
2024.11.04 22:31:42.131647 [ 1407 ] {} <Fatal> BaseDaemon: 21. ? @ 0x00007fa88f086850
2024.11.04 22:31:42.131675 [ 1407 ] {} <Fatal> BaseDaemon: Integrity check of the executable skipped because the reference checksum could not be read.
2024.11.04 22:31:42.448841 [ 1407 ] {} <Fatal> BaseDaemon: This ClickHouse version is not official and should be upgraded to the official build.
2024.11.04 22:31:42.449075 [ 1407 ] {} <Fatal> BaseDaemon: Changed settings: receive_timeout = 10., receive_data_timeout_ms = 10000, allow_suspicious_low_cardinality_types = true, log_queries = true, table_function_remote_max_addresses = 200, max_execution_time = 10., join_algorithm = 'hash', max_memory_usage = 10000000000, optimize_read_in_order = true, allow_introspection_functions = true, allow_experimental_analyzer = true
2024.11.04 22:31:49.990635 [ 158 ] {} <Fatal> Application: Child process was terminated by signal 6.
```

### LINKED ISSUE #71507: Paste join with Group by logical error
**Describe the bug**
A query with two `PASTE JOIN` and `GROUP BY` gives a logical error on the latest head.

**How to reproduce**
Run:
https://fiddle.clickhouse.com/1afb1b88-c0eb-49e4-b768-cc9126813236

Stack trace:
```
<Fatal> : Logical error: 'Invalid number of columns in chunk pushed to OutputPort. Expected 2, found 3
Header: __table2.c0 UInt8 UInt8(size = 0), __table4.c0 Int32 Int32(size = 0)
Chunk:  Int32(size = 1) Const(size = 1, UInt8(size = 1)) Int32(size = 1)
'.
<Fatal> : Stack trace (when copying this message, always include the lines below):

0. ./contrib/llvm-project/libcxx/include/exception:141: Poco::Exception::Exception(String const&, int) @ 0x000000003510fb74
1. src/Common/Exception.cpp:109: DB::Exception::Exception(DB::Exception::MessageMasked&&, int, bool) @ 0x000000001a5746a9
2. DB::Exception::Exception(PreformattedMessage&&, int) @ 0x000000000ae00d85
3. ./src/Common/Exception.h:129: DB::Exception::Exception<unsigned long, unsigned long, String, String>(int, FormatStringHelperImpl<std::type_identity<unsigned long>::type, std::type_identity<unsigned long>::type, std::type_identity<String>::type, std::type_identity<String>::type>, unsigned long&&, unsigned long&&, String&&, String&&) @ 0x0000000028bd99a6
4. ./src/Processors/Port.h:415: DB::IMergingTransformBase::prepare() @ 0x000000002e815ea4
5. src/Processors/Executors/ExecutingGraph.cpp:277: DB::ExecutingGraph::updateNode(unsigned long, std::queue<DB::ExecutingGraph::Node*, std::deque<DB::ExecutingGraph::Node*, AllocatorWithMemoryTracking<DB::ExecutingGraph::Node*>>>&, std::queue<DB::ExecutingGraph::Node*, std::deque<DB::ExecutingGraph::Node*, AllocatorWithMemoryTracking<DB::ExecutingGraph::Node*>>>&) @ 0x000000002df71a0a
6. src/Processors/Executors/PipelineExecutor.cpp:308: DB::PipelineExecutor::executeStepImpl(unsigned long, std::atomic<bool>*) @ 0x000000002df65d21
7. src/Processors/Executors/PipelineExecutor.cpp:255: void std::__function::__policy_invoker<void ()>::__call_impl<std::__function::__default_alloc_func<DB::PipelineExecutor::spawnThreads()::$_0, void ()>>(std::__function::__policy_storage const*) @ 0x000000002df67c15
8. ./contrib/llvm-project/libcxx/include/__functional/function.h:848: ? @ 0x000000001a7436f0
9. ./contrib/llvm-project/libcxx/include/__functional/invoke.h:359: ? @ 0x000000001a74f3a1
10. ./contrib/llvm-project/libcxx/include/__functional/function.h:848: ? @ 0x000000001a73e852
11. ./contrib/llvm-project/libcxx/include/__functional/invoke.h:359: ? @ 0x000000001a74ac55
12. asan_thread_start(void*) @ 0x000000000adb5059
13. ? @ 0x00007ffff7ca1e2e
14. ? @ 0x00007ffff7d33a4c
```

### PR DESCRIPTION
**Pull request by vdimir**
### Changelog category (leave one):

- New Feature

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):

* Added an option to select the side of the join that will act as the inner (build) table in the query plan. This is controlled by `query_plan_join_swap_table`, which can be set to `auto`. In this mode, ClickHouse will try to choose the table with the smallest number of rows.

Close https://github.com/ClickHouse/ClickHouse/issues/71454, close https://github.com/ClickHouse/ClickHouse/issues/71507

### Documentation entry for user-facing changes

- [x] Documentation is written (mandatory for new features)

### CODE REVIEW

**Comment by robot-ch-test-poll2** [comment]
*This is an automated comment for commit 4d973cf7b16dd6d0e5bf80889c29d3b2d552fdf6 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/71577/4d973cf7b16dd6d0e5bf80889c29d3b2d552fdf6/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Builds</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/71573/4b0d3fed4cbee54a4c69f5f6d7958b41aae60a01/builds/report.html">✅ success</a></td></tr>
<tr><td>ClickBench</td><td>Runs [ClickBench](https://github.com/ClickHouse/ClickBench/) with instant-attach table</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/b0393274f757f3a1e896ac2946483310ad1e7b4c/clickbench__aarch64_.html">✅ success</a></td></tr>
<tr><td>Compatibility check</td><td>Checks that <b>clickhouse</b> binary runs on distributions with old libc versions. If it fails, ask a maintainer for help</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/b0393274f757f3a1e896ac2946483310ad1e7b4c/compatibility_check__aarch64_.html">✅ success</a></td></tr>
<tr><td>Fast test</td><td>Normally this is the first check that is ran for a PR. It builds ClickHouse and runs most of <a href="https://clickhouse.com/docs/en/development/tests#functional-tests">stateless functional tests</a>, omitting some. If it fails, further checks are not started until it is fixed. Look at the report to see which tests fail, then reproduce the failure locally as described <a href="https://clickhouse.com/docs/en/development/tests#functional-test-locally">here</a></td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/0094a9df40075fd50d513d6dfa7b490b53dafc91/fast_test.html">✅ success</a></td></tr>
<tr><td>Flaky tests</td><td>Checks if new added or modified tests are flaky by running them repeatedly, in parallel, with more randomization. Functional tests are run 100 times with address sanitizer, and additional randomization of thread scheduling. Integration tests are run up to 10 times. If at least once a new test has failed, or was too long, this check will be red. We don't allow flaky tests, read <a href="https://clickhouse.com/blog/decorating-a-christmas-tree-with-the-help-of-flaky-tests/">the doc</a></td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/71573/4b0d3fed4cbee54a4c69f5f6d7958b41aae60a01/integration_tests_flaky_check__asan_.html">✅ success</a></td></tr>
<tr><td>Install packages</td><td>Checks that the built packages are installable in a clear environment</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/b0393274f757f3a1e896ac2946483310ad1e7b4c/install_packages__aarch64_.html">✅ success</a></td></tr>
<tr><td>Unit tests</td><td>Runs the unit tests for different release types</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/b0393274f757f3a1e896ac2946483310ad1e7b4c/unit_tests__asan_.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

File: src/Core/Settings.cpp [hunk 1]
```diff
@@ -1918,6 +1918,13 @@ See also:
 For single JOIN in case of identifier ambiguity prefer left table
 )", IMPORTANT) \
     \
+DECLARE(BoolAuto, query_plan_join_swap_table, {}, R"(
```
Comment (vdimir) [inline src/Core/Settings.cpp:1921]: cc @devcrafter

Comment (devcrafter) [reply src/Core/Settings.cpp:1921]: @vdimir Honestly, the setting looks weird. IMHO, the setting should just enable or disable the automatic mode to choose the table. But to use this setting for swapping tables in JOIN clause - looks weird - user should just rewrite the query itself.

Comment (vdimir) [reply src/Core/Settings.cpp:1921]: It's also useful for testing, as I want to ensure that any pipeline we build is valid and can be executed. 

Another reason is that when the automatically inferred side is incorrect, and the user need an ad-hoc solution to specify the correct one (however, as you mentioned, rewriting is also an option and should work for most use cases).

But in ideal case the setting is used only in tests and rarely by user set to `false` to have legacy behavior.

**Comment by vdimir** [comment]
> Fast test — fail: 207, passed: 5652, skipped: 1380 [Details](https://s3.amazonaws.com/clickhouse-test-reports/71577/e29fc6ee63365098313998cba6bfbf80b7978da0/fast_test.html)

Conflict resolution did not go well

**Comment by vdimir** [comment]
> Integration tests (tsan) [1/6] — fail: 1, passed: 526 [Details](https://s3.amazonaws.com/clickhouse-test-reports/71577/62789dab310a2af861a5fc6e7e893c5095f8a62a/integration_tests__tsan__[1_6].html)

`test_backup_restore_on_cluster/test_cancel_backup.py::test_error_leaves_no_trash`

> Stress test/Upgrade check

https://github.com/ClickHouse/ClickHouse/issues/72248

**Comment by vdimir** [comment]
> Integration tests (tsan) [1/6] — fail: 1, passed: 526 [Details](https://s3.amazonaws.com/clickhouse-test-reports/71577/c10d68c7a0ab4413e96ddb8aafd38849d7927835/integration_tests__tsan__[1_6].html)

`#72391`

**Comment by ppakawatk** [comment]
Hi, I found a potential bug on this feature, and reported it two weeks ago. Just wanted to follow up and see if anyone has taken a look at [this issue](https://github.com/ClickHouse/ClickHouse/issues/78441)?

**Pull request merged**